Core/Config.cpp

```diff
@@ -443,9 +435,7 @@ static ConfigSetting graphicsSettings[] = {
 #endif
 
 	// TODO: Replace these settings with a list of options
-	ConfigSetting("PartialStretch", &g_Config.bPartialStretch, &DefaultPartialStretch, true, true),
```

Comment:
Yeah, I think I just forgot about this since the blackberry default was set at different place, will change it along the rest after I get some sleep ~ busy part of the year @_@.